index.js

```diff
@@ -75,8 +82,41 @@ module.exports = function (source, map) {
     }
   });
 
-  function relativePath(fullPath) {
-    return path.relative(rootPath, fullPath);
+  function getAliases (options) {
+    // TODO: Investigate if there is a way to leverage
+    // the Webpack API to get all the aliases
+    var resolveAliases = options.resolve && options.resolve.alias;
+    var resolveLoaderAliases = options.resolveLoader && options.resolveLoader.alias;
+    return Object.assign({}, resolveAliases || {}, resolveLoaderAliases || {});
```

Comment:
Oh my bad! So used to writing ES6 nowadays ;) Any plans btw to migrate to using Babel or alike? For now, I'd suggest shimming this using [lodash.assign](https://www.npmjs.com/package/lodash.assign).